### ISSUE TITLE
Information Systems - Long chapter title runs off page

### ISSUE THREAD

**Issue by caylaeagon** [openstax/cnx-recipes#5424]
# Brief Description of Scenario

Long chapter titles are running off the margin of the chapter splash page.
<img width="953" alt="Screenshot 2024-01-25 at 3 29 20 PM" src="https://github.com/openstax/cnx-recipes/assets/99201272/60359c03-5bfa-4cf6-8227-a3f4dd8e7dbe">

# Pattern of Occurrence 

There is only once instance available for testing.

# Acceptance Criteria

Long chapter titles should have a line break to fit on the page
  
# Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

# Location(s)
- **osbooks repo:** https://github.com/openstax/osbooks-foundations-of-information-systems/tree/design-chapters
- **collection slug:** foundations-of-information-systems
- **ref/branch:** design-chapters
- **chapter/section:** Chapter intro/splash page
- **surrounding text:**

**Comment by openstaxalina** [comment]
putting in c.c. - Info Sys Design Chapter & Survey are due to WW after that sprint (Feb 12)

**Comment by wilkus27** [comment]
### Information for testers
Changes has an impact on all books from Cardboard.

**Comment by MartaSztuka** [comment]
@wilkus27 can we increase margin?

![Image](https://github.com/openstax/cnx-recipes/assets/47241114/b1d13e4c-bb2f-48bd-b9db-5a82c85b108e)

**Comment by wilkus27** [comment]
@MartaSztuka it's ready for retest.

Just for information, this problem is related to our functions `font-scale` and `line-height`, which were created a few years ago at the beggining of the project. For some reason, for value `3` line height is smaller then the font size. We should probably fix this in the future, but it will have an impact on all books. As I checked, in other headers of this size it was fixed by setting `font-scale` (instead of line-height) for line height, so now font size and line height are the same.

cc @lwtchu

**Comment by lwtchu** [comment]
@wilkus27 let's not change it for now - since this is just for testing the template we'll see what folks say after we send for review

**Comment by Stackblocks** [comment]
After the update the title is breaking properly and the descender on the `g` in "emerging" and the riser on the `l` in "value" aren't colliding anymore.

![Image](https://github.com/openstax/cnx-recipes/assets/12678235/e9b786fe-856d-4bb8-a89d-bff9110e8192)



![Image](https://github.com/openstax/cnx-recipes/assets/12678235/8b2801c1-8477-43b4-9ea9-63e6872213b3)

**Comment by wilkus27** [comment]
@lwtchu Sorry, I made a change before your comment. If we decide to go back to the previous version I'll revert the change in a separate PR.
